### ISSUE TITLE
Installer boot: CSM support enabled getting black screen/CSM disabled results in garbled graphics

### ISSUE THREAD

**Issue by drinvincibilis** [racka98/Lenovo-Thinkpad-T450-T450s-Hackintosh-Guide-Opencore#9]
Hi, thanks for sharing your efforts here. I followed your recipe thoroughly assuming making macOS installer inside the Virtual Box macOS VM.
Unfortunately lack of success with booting the installer:

1. Enabled CSM support in BIOS - this results in hanging completely with black screen (lights on) right after choosing to boot from the USB installer stick
2. Disabled CSM support in BIOS (mentioned here: https://github.com/jsassu20/Lenovo-T450-Catalina-OpenCore) - this gives me console mode boot but then when entering graphics the screen is totally garbled.

Any suggestions would be much appreciated.

**Platform:**
System Manufacturer	LENOVO	
System Model	20BWA04FPB	
System Type	x64-based PC	
System SKU	LENOVO_MT_20BW_BU_Think_FM_ThinkPad T450s	
Processor	Intel(R) Core(TM) i7-5600U CPU @ 2.60GHz, 2594 Mhz, 2 Core(s), 4 Logical Processor(s)	
BIOS Version/Date	LENOVO JBET73WW (1.37 ), 14/08/2019	
SMBIOS Version	2.7	
Embedded Controller Version	1.04	
BIOS Mode	UEFI	
BaseBoard Manufacturer	LENOVO	
BaseBoard Product	20BWA04FPB	
BaseBoard Version	SDK0E50510 WIN	
Installed Physical Memory (RAM)	12.0 GB	
Total Physical Memory	11.9 GB	
Name	Intel(R) HD Graphics 5500	
Adapter Type	Intel(R) HD Graphics Family, Intel Corporation compatible

**Comment by racka98** [comment]
Follow the guide i linked in my post. CSM support should be enabled. Preferably you should reset bios to default settings and change the settings as recommend.

You are probably getting a black screen because the installer is not correctly made. 

Recheck your usb drive

**Comment by drinvincibilis** [comment]
Hi, Thanks for the feedback. I gave it one more chance and must again confirm that with CSM Enabled the installer doesn't start at all.
However, with CSM Disabled and connected to an external monitor (and boot screen in BIOS set to VGA) I was able to get proper graphics on my laptop LCD screen - totally weird!

I was able to install Big Sur 11.1 successfully. However I found other issues which I would put into other issues here.
